utils/relay-manager.js

```diff
@@ -113,14 +112,8 @@ class RelayManager {
 
     Ignoring multiplication and comparison as that is <10 gas per operation
   */
-  async getGasLimitRefund(_feature, _method, _params, _wallet, _signers, _gasPrice) {
-    let requiredSigsGas = 0;
-    const { contractName } = _feature.constructor;
-    if (contractName === "ApprovedTransfer" || contractName === "RecoveryManager") {
-      requiredSigsGas = 4800;
-    } else if (contractName === "GuardianManager") {
-      requiredSigsGas = 1000;
-    }
+  async getGasLimitRefund(_module, _method, _params, _wallet, _signers, _gasPrice) {
+    const requiredSigsGas = 4800;
```

Comment:
The cost of `getRequiredSignatures` function hasn't changed. Why the change here?

Reply:
Because we had logic to adapt the cost based on the module being called... and it is longer relevant with 1 module.

I don't want to add too much complexity here and start parsing data to determine the called being made so I think that choosing an upper bound to make sure the tests won't fail is ok for now.

I've added a card to revisit the `relay-manager.js` logic in a separate PR.